src/app/components/elements/ChangePassword.jsx

```diff
@@ -72,7 +73,7 @@ class ChangePassword extends React.Component {
             if(onClose) onClose()
             if(resetForm) resetForm()
             notify('Password Updated')
-            window.location = `/login.html#account=${accountName}&msg=passwordupdated`;
+            window.location = pathTo.login() + `#account=${accountName}&msg=passwordupdated`;
```

Comment:
instead of `window.location` should we use [react-router-redux's `push` action](https://github.com/reactjs/react-router-redux#pushlocation-replacelocation-gonumber-goback-goforward)?

Reply:
Yes good point, I'll change it..